cg/resources/rnafusion_bundle_filenames.csv

```diff
@@ -66,13 +66,13 @@
   tag: multiqc-html
 - format: csv
   id: CASEID
-  path: PATHTOCASE/pipeline_info/samplesheet.valid.csv
+  path: PATHTOCASE/workflow_info/samplesheet.valid.csv
   path_index: ~
   step: samplesheet-valid
   tag: samplesheet-valid
 - format: yml
   id: CASEID
-  path: PATHTOCASE/pipeline_info/software_versions.yml
+  path: PATHTOCASE/workflow_info/software_versions.yml
```

Comment:
I think this was just patched back to to "pipeline_info" https://github.com/Clinical-Genomics/cg/pull/2921/files

Will this be fixed with the changes in servers?

Reply:
Good catch! @fevac Could you take a look? IF not ok - let me know and I will switch back.

Reply:
@henrikstranneheim this path has and the other you changed above have to stay as `pipeline_info`. This directory is created by rnafusion and if you rename it it will point to non-existing files and fail during the store step. So please revert back the changes in cg/resources/rnafusion_bundle_filenames.csv 🙏

Reply:
@fevac Thx. Applied your revert.